src/nft/pages/profile/profile.tsx

```diff
@@ -35,12 +35,16 @@ const ProfileContent = () => {
   }, [removeAllMarketplaceWarnings, sellPageState, setListingStatus])
 
   const { account } = useWeb3React()
+  const accountRef = useRef(account)
```

Comment:
why is this needed?

Reply:
The desired outcome in the useEffect was so that when a user changed wallet all the profile states would reset. However just using `account` as the useEffect condition also caused the reset to happen on pageload which prevented me from navigating to the listing page with a pre-selected asset. By adding the check to make for accountRef.current, the expected behaviour now occurs.